Library/Homebrew/keg_relocate.rb

```diff
@@ -5,13 +5,15 @@ class Keg
   PREFIX_PLACEHOLDER = "@@HOMEBREW_PREFIX@@"
   CELLAR_PLACEHOLDER = "@@HOMEBREW_CELLAR@@"
   REPOSITORY_PLACEHOLDER = "@@HOMEBREW_REPOSITORY@@"
+  LIBRARY_PLACEHOLDER = "@@HOMEBREW_LIBRARY@@"
```

Comment:
One last comment. There is no technical benefit that I see to introducing a new placeholder, `@@HOMEBREW_LIBRARY@@`. This one line could be changed to `LIBRARY_PLACEHOLDER = "@@HOMEBREW_REPOSITORY@@/Library", and this PR would function identically without the need for introducing a new placeholder.

Reply:
I think it would be nice to be able to drop `@@HOMEBREW_REPOSITORY@@` entirely, eventually.

Reply:
Every placeholder needs to be supported indefinitely, so that Homebrew can pour old bottles. So there's a [Muller's ratchet](https://en.wikipedia.org/wiki/Muller%27s_ratchet) of complexity at play here. If there's no technical reason to add a new placeholder, I'd err on the side of not adding one. I don't feel super strongly about this, but my preference is not to introduce new placeholders that aren't strictly needed.